
package Protocol;

option optimize_for = LITE_RUNTIME;

enum ClientGateProtocol
{
    PROTOCOL_VERSION            = 0;

    // Gateway: 0-10000
    SERVER_LOGIN_REQUEST        = 0;
    SERVER_LOGIN_RESPONSE       = 1;

    USER_DISCONNECTED           = 2;

    // Client: 10001-20000
    CLIENT_LOGIN_REQUEST        = 10001;
    CLIENT_LOGIN_RESPONSE       = 10002;
}
